test/rest.browser.test.js

```diff
@@ -30,7 +30,7 @@ describe('strong-remoting-rest', function() {
 
   // setup
   beforeEach(function() {
-    objects = RemoteObjects.create();
+    objects = RemoteObjects.create({cors: false});
```

Comment:
The default setting is to enable the built-in CORS middleware with `{ origin: true, credentials: true }`, which triggers deprecation warnings. That was my intention, to poke 2.x users to rework the CORS settings in their application soon.

This applies to our tests too. If I didn't add `{cors: false}` to all the tests, then `npm test` prints a lot of deprecation warnings.

Considering the amount of changes in https://github.com/strongloop/loopback-component-explorer/pull/178 too, perhaps it was not a good idea to add this deprecation message?